File: SlimBundle/Entity/Panier.php
<?php

namespace SlimBundle\Entity;

/**
 * Panier
 */
class Panier
{
    /**
     * @var string
     */
    private $etat;

    /**
     * @var integer
     */
    private $idpanier;

    /**
     * @var \AppBundle\Entity\Utilisateur
     */
    private $idutilisateur;


    /**
     * Set etat
     *
     * @param string $etat
     *
     * @return Panier
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return string
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Get idpanier
     *
     * @return integer
     */
    public function getIdpanier()
    {
        return $this->idpanier;
    }

    /**
     * Set idutilisateur
     *
     * @param \AppBundle\Entity\Utilisateur $idutilisateur
     *
     * @return Panier
     */
    public function setIdutilisateur(\AppBundle\Entity\Utilisateur $idutilisateur = null)
    {
        $this->idutilisateur = $idutilisateur;

        return $this;
    }

    /**
     * Get idutilisateur
     *
     * @return \AppBundle\Entity\Utilisateur
     */
    public function getIdutilisateur()
    {
        return $this->idutilisateur;
    }
}

File: mobileintegration/CodenameOneConsommationWebService/src/com/mycompany/Entite/panier.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

/**
 *
 * @author Hichem
 */
public class panier {
   private int idpanier;
    private int idutilisateur;
   private int etat ;
   

    @Override
    public String toString() {
        return "panier{" + "idpanier=" + idpanier + ", idutilisateur=" + idutilisateur + ", etat=" + etat + '}';
    }

    public panier(int idpanier, int idutilisateur, int etat) {
        this.idpanier = idpanier;
        this.idutilisateur = idutilisateur;
        this.etat = etat;
    }
    public panier( int idutilisateur, int etat) {
       // this.idpanier = idpanier;
        this.idutilisateur = idutilisateur;
        this.etat = etat;
    }

    public panier() {
    }

    public int getIdpanier() {
        return idpanier;
    }

    public void setIdpanier(int idpanier) {
        this.idpanier = idpanier;
    }

    public int getIdutilisateur() {
        return idutilisateur;
    }

    public void setIdutilisateur(int idutilisateur) {
        this.idutilisateur = idutilisateur;
    }

    public int getEtat() {
        return etat;
    }

    public void setEtat(int etat) {
        this.etat = etat;
    }
    
    
}
